src/frontend/src/lib/derived/address.derived.ts

```diff
@@ -1,38 +1,32 @@
-import { BTC_MAINNET_TOKEN_ID } from '$env/tokens.btc.env';
-import { ETHEREUM_TOKEN_ID } from '$env/tokens.env';
-import { addressStore, type OptionAddressData } from '$lib/stores/address.store';
-import type { OptionBtcAddress, OptionEthAddress } from '$lib/types/address';
+import { btcAddressMainnetStore, ethAddressStore } from '$lib/stores/address.store';
+import type {
+	BtcAddress,
+	EthAddress,
+	OptionBtcAddress,
+	OptionEthAddress
+} from '$lib/types/address';
 import { mapAddress } from '$lib/utils/address.utils';
 import { isNullish } from '@dfinity/utils';
 import { derived, type Readable } from 'svelte/store';
 
-export const addressNotLoaded: Readable<boolean> = derived([addressStore], ([$addressStore]) =>
-	isNullish($addressStore)
-);
-
-const btcAddressData: Readable<OptionAddressData> = derived(
-	[addressStore],
-	([$addressStore]) => $addressStore?.[BTC_MAINNET_TOKEN_ID]
+export const addressNotLoaded: Readable<boolean> = derived(
```

Comment:
You rename it right after this PR or we should add a TODO? ah or maybe it gonna become `addressesNotLoad` with both eth and bitcoin, that's possible.

Reply:
i rename it in the next PR, but i will add a TODO either way